portal-api/src/v1/controllers/feedback.controller.js

```diff
@@ -58,6 +58,10 @@ exports.create = async (req, res) => {
     submittedBy
   } = modifiedFeedback;
 
+  if (submittedBy.username === null) {
+    submittedBy.username = 'N/A - not logged in';
```

Comment:
is this what James asked for? `N\A - not logged in` ?

Reply:
no, I added this because it breaks the template if passing null

Reply:
Can we leave it as `N\A` only?